test/lib/sql.js

```diff
@@ -0,0 +1,203 @@
+
+const { expect } = require('chai');
+const SQL = require('../../lib/sql');
+const TABLE_NAME = 'test_table';
+
+describe('SQL', () => {
+	describe('Testing find', () => {
+		const expectedQuery = 'select * from `test_table` where `id` = 1';
+		it('Should find by id if parameter is {id: 1}', () => {
+			let error, query;
+			try {
+				const sql = new SQL(TABLE_NAME);
+				query = sql.find({id: 1});
+			} catch (e) {
+				error = e;
+			}
+			expect(query.toString()).to.be.equal(expectedQuery);
+			expect(error).to.be.undefined;
+		});
+		it('Should find by id if parameter is (`id`, 1)', () => {
+			let error, query;
+			try {
+				const sql = new SQL(TABLE_NAME);
+				query = sql.find('id', 1);
+			} catch (e) {
+				error = e;
+			}
+			expect(query.toString()).to.be.equal(expectedQuery);
+			expect(error).to.be.undefined;
+		});
+		it('Should find by id if parameter is (`id`, `=`, 1)', () => {
+			let error, query;
+			try {
+				const sql = new SQL(TABLE_NAME);
+				query = sql.find('id', '=', 1);
+			} catch (e) {
+				error = e;
+			}
+			expect(query.toString()).to.be.equal(expectedQuery);
+			expect(error).to.be.undefined;
+		});
+		it('Should throw an error if function has more then 3 argument', () => {
+			const sql = new SQL(TABLE_NAME);
+			let error;
+			try {
+				sql.find('id', '=', 1, 'extra');
+			} catch(e) {
+				error = e;
+			}
+			expect(error).to.be.instanceOf(Error);
```

Comment:
isn't better to use something like `expect(....). toThrowError` and assert also the message. I'm sure `chai` has a thing for that.